src/services/githubServices.ts

```diff
@@ -0,0 +1,203 @@
+import { Octokit } from "@octokit/core";
+import { Repository } from "../types/types";
+
+const TOKEN = process.env.GITHUB_TOKEN;
+const octokit = new Octokit({ auth: TOKEN });
+
+/**
+ * Gets the specified branch reference.
+ * @param {Repository} params - The owner and repository name.
+ * @param {string} branchName - The name of the branch.
+ * @returns {Promise<any>} The response from the GitHub API.
+ */
```

Comment:
This seems a lot like a jsdoc comment. Also it does not describe params correctly. 

Here is an example of proper comment (fix all comments in the project before next review): 

```suggestion
/**
 * Gets the specified branch reference.
 * 
 * @param params - The owner and repository name.
 * @param params.owner - The owner of the repository.
 * @param params.repository - The name of the repository.
 * @param branchName - The name of the branch..
 * @returns The response from the GitHub API.
 */
```